### PR TITLE
Add legit and malicious/sus indicators

### PR DESCRIPTION
**Pull request by cmitchelltudor**
Two .csv files for Turkiye earthquake related domains. One file is legit; other is malicious/sus

### CODE REVIEW

**Review by reneeburton — CHANGES_REQUESTED** [review]
you need to add something to the README top file. we'll come back in and add a blog link later. but also there is something wrong with teh last line of the file.

File: turkiye/turkiye_earthquake_malicious_suspicious_iocs.csv [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+domain,sanliurfakoltukyikama.online,phishing,
+domain,xn--depremyardm-8zb.com,phishing,
+domain,donateturkey.world,phishing,
+domain,yardim.b-cdn.net,phishing,
+domain,afad-kizilay.yardimbagis.net,phishing,
+domain,depremkontrol.com.tr,phishing,
+domain,depremproje.com,phishing,
+domain,depremacilyardim.com,phishing,
+domain,deprem.link,phishing,
+,yardim-yap.net,phishing,
```
Comment (reneeburton) [inline turkiye/turkiye_earthquake_malicious_suspicious_iocs.csv:169]: something is wrong here

File: turkiye/turkiye_earthquake_legitimate_iocs.csv.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+domain,yikilanbinalar.com.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,enkazbildirim.org,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,howtohelpturkey-syriaearthquakevictims.org,legitimate,
+domain,hepsiemlak.com,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,kizilay.org.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,misafirimol.trair.com.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,depremihtiyac.com,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,akut.org.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,adresdeprem.com,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,earthquaketurkey.com,legitimate,
```
Comment (reneeburton) [inline turkiye/turkiye_earthquake_legitimate_iocs.csv.csv:11]: why do we know this is legit? just older registration?

File: turkiye/turkiye_earthquake_legitimate_iocs.csv.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
+﻿type,indicator,classification,references
+domain,yikilanbinalar.com.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,enkazbildirim.org,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,howtohelpturkey-syriaearthquakevictims.org,legitimate,
```
Comment (reneeburton) [inline turkiye/turkiye_earthquake_legitimate_iocs.csv.csv:4]: why do we know this is legit?

File: turkiye/turkiye_earthquake_legitimate_iocs.csv.csv (hunk 2 above)
Comment (cmitchelltudor) [reply turkiye/turkiye_earthquake_legitimate_iocs.csv.csv:11]: "Recently registered domain on 2023-02-08. Domain contains news related to the earthquake which happened in Turkey on February 2023. No links or references to funding or anything which could lead to identifying suspicious behaviour at the time of the investigation (2023-02-10)."

File: turkiye/turkiye_earthquake_legitimate_iocs.csv.csv (hunk 3 above)
Comment (cmitchelltudor) [reply turkiye/turkiye_earthquake_legitimate_iocs.csv.csv:4]: "Domain contains information about how can people can support the people of Turkey and Syria which were hit by the earthquake on February 2023. It contains links to legitimate funding initiatives."

File: README.md [hunk 4]
```diff
@@ -1,30 +1,45 @@
...
+blocked by other vendors due to their new registration or other automated analytics.
+
+## turkiye
+This folder contains IOCs related to the earthquake that struck Türkiye and parts of Syria in early February, 2023.
+The majority of the content is based on Infoblox internal analytics and validation analysis, though some OSINT is also
+included.  Our references should clearly indicate indicators that originated in OSINT.
+
+The file `turkiye_earthquake_legitimate_iocs.csv` contains malicious and suspicious IOCs that can cause harm to
+businesses and innocent users. Infoblox recommends blocking traffic from network indicators described in this file.
+
```
Comment (vbopp) [inline README.md:36]: should the description be: ...contains confirmed indicators that, a the time of review, were not evidently associated with malicious or suspicious activity. The related domains did not show indications of hosted malware or fraudulent behavior.

Comment (vbopp) [reply README.md:36]: file should just have the legitimate indicators, right?

File: turkiye/turkiye_earthquake_legitimate_iocs.csv [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+domain,yikilanbinalar.com.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,enkazbildirim.org,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,howtohelpturkey-syriaearthquakevictims.org,legitimate,
+domain,hepsiemlak.com,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,kizilay.org.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,misafirimol.trair.com.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,depremihtiyac.com,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,akut.org.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,adresdeprem.com,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,earthquaketurkey.com,legitimate,
```
Comment (vbopp) [inline turkiye/turkiye_earthquake_legitimate_iocs.csv:11]: needs a reference?

File: turkiye/turkiye_earthquake_legitimate_iocs.csv [hunk 6]
```diff
@@ -0,0 +1,25 @@
+﻿type,indicator,classification,references
+domain,yikilanbinalar.com.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,enkazbildirim.org,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,howtohelpturkey-syriaearthquakevictims.org,legitimate,
```
Comment (vbopp) [inline turkiye/turkiye_earthquake_legitimate_iocs.csv:4]: needs a reference?

File: README.md (hunk 4 above)
Comment (cmitchelltudor) [reply README.md:36]: oops, good catch!

File: turkiye/turkiye_earthquake_legitimate_iocs.csv.csv (hunk 2 above)
Comment (reneeburton) [reply turkiye/turkiye_earthquake_legitimate_iocs.csv.csv]: ok

File: turkiye/turkiye_earthquake_legitimate_iocs.csv.csv (hunk 3 above)
Comment (reneeburton) [reply turkiye/turkiye_earthquake_legitimate_iocs.csv.csv]: ok

**Review by reneeburton — APPROVED** [review]

**Pull request merged**